Controllers/BorrowController.js

```diff
@@ -0,0 +1,120 @@
+const Borrow = require("../Models/BorrowModel");
+const Book = require("../Models/BookModel");
+const Student = require("../Models/StudentModel");
+const { DateDifference } = require("../utils/DateDiff");
+const DefinedError = require("../Middleware/DefinedError");
+const { errHandle } = require("../Middleware/errHandle");
+
+const DuePeriod = 14;
```

Comment:
constants can be defined in a separate file and be used here. better coding practice